bigbluebutton-html5/imports/api/users/server/methods/setMobileUser.js

```diff
@@ -1,27 +1,18 @@
 import { check } from 'meteor/check';
 import Logger from '/imports/startup/server/logger';
+import setMobile from '../modifiers/setMobile';
 import { extractCredentials } from '/imports/api/common/server/helpers';
-import RedisPubSub from '/imports/startup/server/redis';
 
-export default async function setMobileUser() {
+export default function setMobileUser() {
   try {
-    const REDIS_CONFIG = Meteor.settings.private.redis;
```

Comment:
removed lines are part of https://github.com/bigbluebutton/bigbluebutton/pull/17496